library/swift/src/RequestHeaders.swift

```diff
@@ -2,7 +2,7 @@ import Foundation
 
 /// Headers representing an outbound request.
 @objcMembers
-public final class RequestHeaders: Headers {
+public class RequestHeaders: Headers {
```

Comment:
No need for `final`? 

(`RequestTrailers` is `final` btw)

Reply:
`GRPCRequestHeaders` subclasses this, so we can't use `final` here

Reply:
For context, I had considered replicating `RequestHeaders` entirely for gRPC, but that would be problematic for filters if we're passing a `<T: RequestHeaders>` into filters, since gRPC requests wouldn't conform to that. Granted, we may very well be doing that at a much lower layer in which case this wouldn't be a problem, but this approach seemed like the least overhead for now

Reply:
Ah